src/libelektra/mount.c

```diff
@@ -203,7 +203,96 @@ int elektraMountDefault (KDB *kdb, KeySet *modules, Key *errorKey)
 
 	return 0;
 }
+int elektraMountGlobals(KDB *kdb, KeySet *keys, KeySet *modules, Key *errorKey)
+{
+	Key *root;
+	root=ksLookupByName(keys, "system/elektra/globalplugins", 0);
+	if(!root)
+	{
+#if DEBUG && VERBOSE
+		printf("no global configuration exists\n");
+#endif
+		ksDel(keys);
+		return 0;
+	}
+	KeySet *global = ksCut(keys, root);
+	Key *cur;
+	KeySet *referencePlugins = ksNew(0, KS_END);
+	while((cur = ksNext(global)) != NULL)
+	{
+		if(keyRel(root, cur) != 1)
+			continue;
```

Comment:
shouldn't you abort then? (When a key is not below the root anymore, no other will follow afterwards)